src/lem/symbol.rs

```diff
@@ -1,9 +1,18 @@
-#[derive(Clone, PartialEq, Eq, Hash)]
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
 pub enum Symbol {
     Sym(Vec<String>),
     Key(Vec<String>),
 }
```

Comment:
What is the benefit of not just using Lurk's `Symbol`? It seems like this would simultaneously spare you from having to implement something 'similar' and also avoid eventual pain from incompatibility.

Reply:
This `symbol.rs` file will disappear. This shim is temporary, while the mega PR isn't merged